src/features/Application/ManageApplicationContainer.tsx

```diff
@@ -147,74 +149,72 @@ class ManageApplicationContainer extends React.Component<
         this.setState({ mode: ManageApplicationModes.CREATE });
       }
     }
+    this.setState({ loaded: true });
   }
 
   public render() {
-    const { mode, hackerDetails, submitted, pageNumber } = this.state;
-    return submitted ? (
-      <Redirect to={FrontendRoute.HOME_PAGE} />
-    ) : (
-      // <HorizontalSpacer paddingLeft={'20%'}>
-      <MaxWidthBox m={'auto'} maxWidth={'500px'}>
-        {/* <Sidebar
-            currentPage="Application"
-            status={this.state.hackerDetails.status}
-            confirmed={true}
-          /> */}
-        <BackgroundImage
-          right={'10%'}
-          top={'178px'}
-          src={Drone}
-          imgHeight={'133px'}
-          position={'fixed' as 'fixed'}
-        />
-        <BackgroundImage
-          left={'5%'}
-          bottom={'5%'}
-          src={Bulby}
-          imgHeight={'290px'}
-          position={'fixed' as 'fixed'}
-        />
-        <Helmet>
-          <title>
-            {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'}{' '}
-            Application | {CONSTANTS.HACKATHON_NAME}
-          </title>
-        </Helmet>
-        <MaxWidthBox maxWidth={'500px'} m={'auto'}>
-          <H1
-            color={theme.colors.red}
-            fontSize={'30px'}
-            textAlign={'left'}
-            marginTop={'0px'}
-            marginBottom={'20px'}
-            marginLeft={'0px'}
-            paddingBottom={'20px'}
-            paddingTop={'70px'}
-          >
-            {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'} your
-            Application
-          </H1>
-          <FormDescription>{CONSTANTS.REQUIRED_DESCRIPTION}</FormDescription>
+    const { mode, hackerDetails, submitted, pageNumber, loaded } = this.state;
+    return loaded ? (
+      // If application creation deadline has passed or form is submitted, return user to the home page
+      (Date.now() > 1578070799000 && mode === ManageApplicationModes.CREATE) ||
```

Comment:
what's this date number? can you write a comment for what date it is for future reference?